src/main/java/org/team199/robot2020/RobotContainer.java

```diff
@@ -112,7 +119,8 @@ private void configureButtonBindingsLeftJoy() {
         new JoystickButton(leftJoy, Constants.OI.LeftJoy.kToggleDriveModeButton).whenPressed(new InstantCommand(
                 () -> SmartDashboard.putBoolean("Arcade Drive", !SmartDashboard.getBoolean("Arcade Drive", false))));
 
-        // characterize drive button
+        ToggleCamera.setCamera(2);
```

Comment:
Setting the default camera should happen in the subsystem that manages the cameras.